CMSgov/design-system
# [WNMGDS-753] Update styles for focus buttons

## Summary
- Set the focus styles to use the same text and background color as the hover styles for each button variation when the focus styles are not implemented 

### Changed
- Changed variable values to use the correct colors for focus buttons

## How to test
- Run the doc site locally and see that the focus styles are visible when you tab through the buttons
- Set the focus style flag to true to make sure when focus styles are set they work as expected
